examples/ice_advected_by_anticyclone.jl

```diff
@@ -0,0 +1,193 @@
+# # Sea ice advected by an atmospheric anticyclone
+#
+#
+#
+#
+
+using Oceananigans
+using Oceananigans.Units
+using ClimaSeaIce
+using Printf
+using ClimaSeaIce.SeaIceDynamics
+
+# The experiment found in the paper: 
+# Simulating Linear Kinematic Features in Viscous-Plastic Sea Ice Models 
+# on Quadrilateral and Triangular Grids With Different Variable Staggering
+
+arch = CPU()
+
+L  = 512kilometers
+𝓋ₒ = 0.01 # m / s maximum ocean speed
+𝓋ₐ = 30.0 # m / s maximum atmospheric speed modifier
+Cᴰ = 1.2e-3 # Atmosphere - sea ice drag coefficient
+ρₐ = 1.3 # kg/m³
+
+# 2 km domain
+grid = RectilinearGrid(arch;
+                       size = (256, 256), 
+                          x = (0, L), 
+                          y = (0, L), 
+                   topology = (Bounded, Bounded, Flat))
+
+#####
+##### Setup atmospheric and oceanic forcing
+#####
+
+# Constant ocean velocities corresponding to a cyclonic eddy
+Uₒ = XFaceField(grid)
+Vₒ = YFaceField(grid)
+
+set!(Uₒ, (x, y) -> 𝓋ₒ * (2y - L) / L)
+set!(Vₒ, (x, y) -> 𝓋ₒ * (L - 2x) / L)
+
+Uₐ = XFaceField(grid)
+Vₐ = YFaceField(grid)
+
+# Atmosphere - sea ice stress
+τᵤ = Field(ρₐ * Cᴰ * sqrt(Uₐ^2 + Vₐ^2) * Uₐ)
+τᵥ = Field(ρₐ * Cᴰ * sqrt(Uₐ^2 + Vₐ^2) * Vₐ)
+
+# Atmospheric velocities corresponding to an anticyclonic eddy moving north-east
+@inline center(t) = 256kilometers + 51.2kilometers * t / 86400
+@inline radius(x, y, t)  = sqrt((x - center(t))^2 + (y - center(t))^2)
+@inline speed(x, y, t)   = 1 / 100 * exp(- radius(x, y, t) / 100kilometers)
+
+@inline ua_time(x, y, t) = - 𝓋ₐ * speed(x, y, t) * (cosd(72) * (x - center(t)) + sind(72) * (y - center(t))) / 1000
+@inline va_time(x, y, t) = - 𝓋ₐ * speed(x, y, t) * (cosd(72) * (y - center(t)) - sind(72) * (x - center(t))) / 1000
+
+# Initialize the stress at time t = 0
+set!(Uₐ, (x, y) -> ua_time(x, y, 0))
+set!(Vₐ, (x, y) -> va_time(x, y, 0))
+compute!(τᵤ)
+compute!(τᵥ)
+
+#####
+##### Numerical details
+#####
+
+# We use an elasto-visco-plastic rheology and WENO seventh order 
+# for advection of h and ℵ
+momentum_solver = ExplicitMomentumSolver(grid)
```

Comment:
it's called "dynamics" below and "solver" here. Does this represent a solver, or physics?

Reply:
eh, it's both. Momentum can be solved implicitly or explicitly and with different rheology. We could call it just `ExplicitSeaIceDynamics` maybe

Reply:
it might be useful to separate the concepts of dynamics and solvers.